pkg/ruler/ruler.go

```diff
@@ -105,6 +105,8 @@ type Config struct {
 	PollInterval time.Duration `yaml:"poll_interval"`
 	// Path to store rule files for prom manager.
 	RulePath string `yaml:"rule_path"`
+	// Default offset for all rule evaluation queries.
+	RuleQueryOffset time.Duration `yaml:"rule_query_offset"`
```

Comment:
Is it better for this to be a per tenant configuration instead?

Reply:
Good point, yes this should be per tenant. Will update.

Reply:
Curious why the evaluation interval is not per tenant as well, is this a bug in Cortex?

Reply:
I am not sure about the reason why as it is a config that was added long time ago. But overall I don't see why it cannot be a per tenant config. But let's leave it as it is.